c2corg_ui/static/js/listswitchbutton.js

```diff
@@ -0,0 +1,48 @@
+goog.provide('app.ListSwitchButtonController');
```

Comment:
These directive/controller seem the same than the ones of the action. Perhaps you could avoid duplicating code by switching the template depending on some parameter? See for instance https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/card.js#L24-L25

Or you could group both templates into one and use ``ng-if`` to figure out which layout to use?